app/views/foods/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+  <table class="table table-striped table-hover">
+    <thead>
+      <tr>
```

Comment:
It'd be nice to create a button to add new food, otherwise, the user won't be able to add it.
Here is a suggested image in order to help you.

![Captura de Pantalla 2022-03-08 a la(s) 14 45 49](https://user-images.githubusercontent.com/88604987/157324800-d95b0666-af10-4c59-8c31-0062c18a2534.png)

Reply:
@AlexRS90 thank you for pointing that out 🔍  Fixing it right away 🚗